python/pyspark/sql/connect/column.py

```diff
@@ -54,18 +55,18 @@
 from pyspark.sql.connect.types import pyspark_types_to_proto_types
 
 if TYPE_CHECKING:
-    from pyspark.sql.connect._typing import ColumnOrName, PrimitiveType
+    from pyspark.sql.connect._typing import ColumnOrName, PrimitiveType, WindowSpecType
     from pyspark.sql.connect.client import SparkConnectClient
     import pyspark.sql.connect.proto as proto
 
-JVM_BYTE_MIN = -(1 << 7)
-JVM_BYTE_MAX = (1 << 7) - 1
-JVM_SHORT_MIN = -(1 << 15)
-JVM_SHORT_MAX = (1 << 15) - 1
-JVM_INT_MIN = -(1 << 31)
-JVM_INT_MAX = (1 << 31) - 1
-JVM_LONG_MIN = -(1 << 63)
-JVM_LONG_MAX = (1 << 63) - 1
+JVM_BYTE_MIN: int = -(1 << 7)
+JVM_BYTE_MAX: int = (1 << 7) - 1
+JVM_SHORT_MIN: int = -(1 << 15)
+JVM_SHORT_MAX: int = (1 << 15) - 1
+JVM_INT_MIN: int = -(1 << 31)
+JVM_INT_MAX: int = (1 << 31) - 1
+JVM_LONG_MIN: int = -(1 << 63)
```

Comment:
add the type, otherwise `Cannot determine type of "JVM_LONG_MIN"  [has-type]` in `window.py`